less/patternfly-react.less

```diff
@@ -1,3 +1,7 @@
 /**
   Patternfly React Specific Extensions
 */
+@import "~bootstrap/less/variables";
+@import '~patternfly/dist/less/variables';
```

Comment:
I don't think we want to do this here. The includes for externals should be in a separate file from the inclusion of all the partials. This allows applications to not include the externals (they would likely be doing this already and if so may have already over-ridden some of the variables) but still include just one file from here.

This should have already been setup correctly like the sass structure, unfortunately this must have been missed..

Reply:
The `patternfly-react.less` is on the same "level" as `patternfly-react.scss` which contains similar imports for sass. For this reason, I added these lines here.

Anyway, if the consuming application might decide to not import these, I can introduce new `patternfly-react-deps.less` (or so) and put it there. The consuming code will be responsible to either import this 2nd `.less` or provide its own.

Reply:
The issue with that is that the application would then need to import each of the partial less files rather than a single less file and would need to keep up with any additional files added.

Reply:
The idea was to introduce second `.less` (one) importing all dependencies in a patternfly way. The consuming application can later decide not to import this and handle dependencies on its own.
So just two imports would be needed.

IIUC, in case of sass, the consumer imports whole bundle as it is and can farther adjust using  own styling. Please correct me.

So keeping the imports in the top-most `patternfly-react.less` (means recent PR state) would be same as with the sass. The only difference is, that the `patternfly-react.less` in this PR imports just the required dependencies and not everything else.

@jeff-phillips-18 , what do you propose how to proceed, please?

Reply:
I would prefer that the less setup be made to be like the sass setup. I'm OK with doing that as in a follow up PR though,